.github/workflows/testing.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.4', '1.5']
+        julia-version: ['1.5', '1.6']
```

Comment:
```suggestion
        julia-version:
          - '1.4'
          - '1'
```

Reply:
Don't we want to test against the current LTS, which I *think* is still 1.4, and the latest version, which I think again can be represented as just 1. Unless it actually doesn't work with 1.4? If so, maybe just enforce 1.6 - it'll be the next LTS I think.

Reply:
It fails on 1.4 because DataRegistryUtils requires 1.5 or higher.